pages/api/interest-zones/index.ts

```diff
@@ -3,7 +3,7 @@ import { ZoneValidationError, zoneService } from '../../../services/zone-service
 import type { NextApiRequest, NextApiResponse } from 'next';
 import { withApiAuthRequired} from '@auth0/nextjs-auth0';
 
```

Comment:
please leave some "TODOs" everywhere you've removed the Auth0 security constraints (so as we don't forget about them)